### PR TITLE
Remove GetOpt

### PR DESCRIPTION
**Pull request by follitude**
Since disni has used Cli command line parsing, `mvn install -DskipTests` can not work anymore.

### CODE REVIEW

**Review by patrickstuedi — CHANGES_REQUESTED** [review]
Thanks for this PR! Could you add some description to the parameters, e.g., 

Option addressOption = Option.builder("a").desc("server address").hasArg().build();

Thanks!
-Patrick

File: src/test/java/com/ibm/darpc/examples/client/DaRPCClient.java [hunk 1]
```diff
@@ -183,29 +192,43 @@ public void launch(String[] args) throws Exception {
...
-			if ((char) ch == 'a') {
-				ipAddress = go.optArgGet();
-			} else if ((char) ch == 'k') {
-				loop = Integer.parseInt(go.optArgGet());
-			} else if ((char) ch == 'n') {
-				threadCount = Integer.parseInt(go.optArgGet());
-			} else if ((char) ch == 'm') {
-				String _mode = go.optArgGet();
+		try {
+			CommandLine line = parser.parse(options, Arrays.copyOfRange(args, 0, args.length));
```
Comment (asqasq) [inline src/test/java/com/ibm/darpc/examples/client/DaRPCClient.java]: Why do you do this copy of the array? Can you not just pass args?

File: src/test/java/com/ibm/darpc/examples/server/DaRPCServer.java [hunk 2]
```diff
@@ -61,47 +69,66 @@ public void run() throws Exception{
...
+		options.addOption(recvQueueOption);
+		options.addOption(sendQueueOption);
+		options.addOption(serializedSizeOption);
+		CommandLineParser parser = new DefaultParser();
 
-		GetOpt go = new GetOpt(_args, "a:p:s:da:c:r:s:w:l:");
-		go.optErr = true;
-		int ch = -1;
+		try {
+			CommandLine line = parser.parse(options, Arrays.copyOfRange(args, 0, args.length));
```
Comment (asqasq) [inline src/test/java/com/ibm/darpc/examples/server/DaRPCServer.java]: Why do you do this copy of the array? Can you not just pass args?

**Comment by follitude** [comment]
Done. @patrickstuedi

**Comment by patrickstuedi** [comment]
Great. Thanks!

**Pull request merged**